pennylane/operation.py

```diff
@@ -1135,7 +1135,7 @@ def pow(self, z):
         if z == 0:
             return []
         if z == 1:
-            return [self.__copy__()]
+            return [copy.copy(self)]
```

Comment:
They are exactly the same. `copy.copy()` calls `self.__copy__()`. I changed it because using self.__copy__() raised a pylint error: 
```
Unnecessarily calls dunder method __copy__. Use copy.copy function.pylint(unnecessary-dunder-call)
```